conf_files/pyro_camera.yaml

```diff
@@ -0,0 +1,99 @@
+# Config file for a distributed camera & focuser.
+#
+# Contains a default configuration with a simulated camera and focuser for testing
+# and example configurations for other camera and focusers (commented out).
+#
+# Do not edit this file, instead copy one of the examples to pyto_camera_local.yaml
+# before editing as required.
+#
+# Simulated camera and focusers:
+name: camera.simulator.001
+host: 127.0.0.1
+camera:
+  model: simulator
+  port: /dev/ttyFAKE01
+  set_point: 0
+  filter_type: r2_6
+  focuser:
+    model: simulator
+    port: /dev/ttyFAKE02
+    initial_position: 21706
+    autofocus_keep_files: False
+    autofocus_range:
+      - 50
+      - 250
+    autofocus_step:
+      - 5
+      - 25
+    autofocus_seconds: 1
+    autofocus_size: 500
+directories:
+    base: /var/panoptes
+    images: temp
+    data: data
+#
+# SBIG camera with Birger focuser:
```

Comment:
Do these commented out entries serve a purpose? E.g. do they document something not shown above?

Reply:
Yes, they're templates/examples for real hardware configurations. They don't differ a lot from the uncommented localhost + simulators config but they do illustrate typical values.

Reply:
More documentation makes sense, the question is where to put it. The `name` and `host` entries are the only two here that are specific the distributed camera configuration. The `camera` and `directories` sections, and the `focuser` sub-section, are exactly the same as for a regular `pocs_local.yaml` config. Currently the only documentation for these are in the docstrings of the corresponding classes.